src/components/ebay-education-notice/examples/prominent-icon.marko

```diff
@@ -0,0 +1,13 @@
+import vaultIcon from '<ebay-the-ebay-vault-24-icon>';
+
+<ebay-education-notice ...input a11y-text="ebay vault" icon-variant="prominent" education-icon=vaultIcon>
+    <@title>
+        <div>Notice title</div>
+    </@title>
+    <p>
```

Comment:
The markup in the examples doesn't seem right. It should result in this:

```html
<section class="education-notice" role="region" aria-label="Education" aria-roledescription="Notice">
    <div class="education-notice__header">
        <svg class="icon icon--the-ebay-vault-24" height="24" width="24" aria-label="Education">
            <use href="#icon-the-ebay-vault-24"></use>
        </svg>
        <h3 class="education-notice__title">The Vault</h3>
        <button aria-label="Close notice" class="fake-link education-notice__dismiss">
            <svg aria-hidden="true" class="icon icon--close-16" height="16" width="16">
                <use href="#icon-close-16"></use>
            </svg>
        </button>
    </div>
    <div class="education-notice__main">
        <p>You can now use the Vault to store items like this securely.</p>
    </div>
    <div class="education-notice__footer">
        <button class="fake-link">Learn more button</button>
    </div>
</section>
```

The `p` tags here seem like they are just floating here and this is much more flat than the Skin markup. 
<img width="277" alt="image" src="https://github.com/eBay/ebayui-core/assets/1675667/285cd8a2-8d31-4689-8a47-d32d2561001e">

It results in incorrect spacing as there should be a gap between the description and the CTA.
<img width="1070" alt="image" src="https://github.com/eBay/ebayui-core/assets/1675667/a37d8f37-cb5b-4b93-ae59-0c6a3abd5ef8">

Reply:
This just shows there are multiple <p> elements
The CTA should be usually in the footer.

Reply:
There are no examples with links in the body in the Figma, nor multiple `<p>` tags. Those are not approved use cases for this component. As such, I did not build affordance for those use cases into Skin. In fact, not only does the Figma not have the use case for multiple `<p>` tags, but it actually includes a character limit of 120 characters for the description/detail, which is indicative of a short single paragraph of text.

Building things into ebay ui that do not have corresponding use cases supported in Skin is simply going to break the layout.

Reply:
I removed the second `<p>` in the examples.

That said, we don't have any places where we restrict the character count. We have done `line-clamp` to restrict the line numbers, but not the amount of text. We have this on buttons which have a max char count as well but don't restrict it.

Reply:
Hmm, not sure I agree with design here in terms of trying to forcefully restrict the character count. For web I would prefer things to be more _resilient_, and not have layout breakage even if a user does something they shouldn't do and puts in multiple paragraphs. I think we have some stories to cover these kinds of edge cases in skin (maybe page notice). On web lets encourage wrapping of text where we can easily accommodate it.

Reply:
There might be some confusion here. I didn't actually build anything in Skin to honor the character limit. I wasn't intending ebay ui to honor it either. I only mentioned it here to paint the specific limited use of the component as intended by DS. It would be very odd to allow multiple paragraphs of detailed text here if the original intent was to have a very brief snippet of text: "_is indicative of a short single paragraph of text._"

Reply:
I mean even if you look in the initial example, with multiple paragraphs, it looks fine and not broken. There isn't a lot of spacing between them, but it seems fine.